firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2064,12 +2082,17 @@ class BrowserViewController: UIViewController,
         }
     }
 
-    func presentActionSheet(from view: UIView) {
+    func presentTabsLongPressAction(from view: UIView) {
         guard presentedViewController == nil else { return }
 
         var actions: [[PhotonRowActions]] = []
-        actions.append(getTabToolbarLongPressActionsForModeSwitching())
-        actions.append(getMoreTabToolbarLongPressActions())
+        let toolbarRefactorEnabled = featureFlags.isFeatureEnabled(.toolbarRefactor, checking: .buildOnly)
```

Comment:
Shouldn't we only display the different long press actions if the `toolbarOneTapNewTab` feature is enabled?

Reply:
These long press actions are for the tab tray button. Would you say that falls more under `toolbarOneTapNewTab` instead of `toolbar-refactor-feature`?

Reply:
We will actually have to check that both feature flags are true.